pxr/imaging/hdSt/subdivision.cpp

```diff
@@ -1342,7 +1370,12 @@ HdSt_OsdIndexComputation::_PopulateUniformPrimitiveBuffer(
         : 0;
     VtVec3iArray primitiveParam(numPatches);
     VtVec2iArray edgeIndices(numPatches);
-
+    VtVec3iArray tessFactors(numPatches);
+    
+    float oneFloat = 1.0f;
+    uint32_t one = 0;
+    std::memcpy(&one, &oneFloat, sizeof(oneFloat));
```

Comment:
We might skip initializing the tess factors data buffer when we know we are going to run a PTCS shader